other-metadata.rst

```diff
@@ -94,4 +94,51 @@ this circumstance, and they will not fail for users.
    common, and there is little harm in overspecifying it.
 
 
+.. index:: license
+
+LICENSE
+-------
+:Source: QA
+:Reported: no
+
+The ``LICENSE`` variable must explicitly list licenses for all files
+installed by the package.  If some of the applicable licenses are
+conditional to USE flags, appropriate USE conditionals need to
+be expressed in the variable.
+
+If the package bundles any dependencies that are either installed,
+statically linked or in any other way combined with installed files,
+the licenses of these dependencies need to be listed as well.  This
+is not required when statically linking to dependencies
+installed by separate Gentoo packages.
+
+The licenses for files that are not installed but that are used at build
+time are not listed explicitly.
+
+*Rationale*: the primary purpose of the license support in package
+manager is to provide the users with ability to decide on acceptable
+licenses for their installed systems (and binary packages).  In order
+for this to work effectively, the packages must provide a correct
+and complete license list.
+
+Static linking combines code from multiple packages, potentially covered
+by different licenses.  Listing all of them is the simplest way
+of ensuring that nothing is missed, as well as protecting against wrong
+derivative work licenses stated upstream (i.e. when a less restrictively
+licensed package links to a more restrictively licensed dependency).
+This is enforced for bundled dependencies but not for static linking
```

Comment:
Start a new paragraph here. `\n\nListing of licenses is enforced for bundled dependencies...`